src/app/core.py

```diff
@@ -0,0 +1,469 @@
+"""This file contains the core logic used in the SPDX Online Tools' APP and API"""
+
+import jpype
+import os
+
+from django.utils.datastructures import MultiValueDictKeyError
+from django.core.files.storage import FileSystemStorage
+from django.conf import settings
+
+from json import dumps, loads
+from time import time
+from traceback import format_exc
+from urllib.parse import urljoin
+from app.views import formatToContentType, getFileFormat
```

Comment:
This will be a cyclic dependency kind of logic, views depends on core, core depends on views for some functions

Reply:
Understood.